install/extension_points.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Hi Ingo, ich habe gerade erst gesehen, dass du einen PR erstellt hast. Danke dafür!
Ich habe allerdings ein paar Änderungsvorschläge.
1. Diesen Teil würde ich direkt in der boot.php einfügen.